content/docs/1.1.0/advanced-resources/data-recovery/recover-without-system.md

```diff
@@ -11,19 +11,29 @@ This command gives users the ability to restore a backup to a `raw` image or a `
 
 3. Specify the host path of output file by modifying field `hostpath` of volume `disk-directory`. By default the directory is `/tmp/restore/`.
 
-4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://backupbucket@us-east-1/backupstore?backup=backup-bd326da2c4414b02&volume=volumeexamplename`. Do not delete `''`.
+4. Set the first argument (backup url) by replacing `<BACKUP_URL>`, e.g. `s3://<your-bucket-name>@<your-aws-region>/backupstore?backup=<backup-name>&volume=<volume-name>`. Do not delete `''`.
```

Comment:
`Do not delete ..` is confusing here...

Reply:
It seems no need to mention that. cc @weizhe0422 ?